.github/workflows/testing.yml

```diff
@@ -0,0 +1,71 @@
+name: testing
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - 'master'
+    tags:
+      - '*'
```

Comment:
I would prefer to run tests on any branch and skip duplicating pull request jobs where appropriate. See https://github.com/tarantool/expirationd/pull/55 for example. Okay to leave testing on tags (don't know, whether we need it, but anyway).

Reply:
I am OK with that, fixed.